rero_ils/modules/stats/cli.py

```diff
@@ -50,11 +50,60 @@ def dumps(type):
         pprint(StatsForLibrarian(to_date=arrow.utcnow()).collect(), indent=2)
 
 
+@stats.command('live')
+@click.argument('pid')
+@with_appcontext
+def live(pid):
+    """Extract the stats value for preview.
+
+    :param type: type of statistics can be 'billing' or 'librarian'
+    """
+    from .api.report import StatsReport
+    from ..stats_cfg.api import StatConfiguration
+    cfg = StatConfiguration.get_record_by_pid(pid)
+    if not cfg:
+        click.secho(f'Configuration does not exists.', fg='red')
+    else:
+        from pprint import pprint
+        pprint(StatsReport(cfg).collect())
```

Comment:
Is this correct or for debug ? If you just need to "preview/debug" in console mode, it seems correct (but maybe difficult to read for a human ^^)

Reply:
Yes it is more for debug or diagnostics.